orm/model/ormtable/table.go

```diff
@@ -123,3 +123,11 @@ type Table interface {
 	// ID is the ID of this table within the schema of its FileDescriptor.
 	ID() uint32
 }
+
+type AutoIncrementTable interface {
+	Table
+
+	// InsertWithID inserts the provided entry in the store and returns the newly
+	// generated ID for the message or an error.
+	InsertWithID(ctx context.Context, message proto.Message) (newId uint64, err error)
```

Comment:
Not sure about the name of this function. Alternate names could be `InsertAndGetID`, `InsertAndGetSeq`, `InsertWithSeq`, InsertGetSeq`

Reply:
I'd not use `With` because it seems to suggest that you also need to provide an `ID`.

Fine with every other alternative.

Reply:
So maybe InsertAndGetID?

Reply:
renamed to `InsertReturningID` which is similar to how postgres does it. does that work?

Reply:
Yes!